### ISSUE TITLE
Stress testing of devoxxuk2017 demo

### ISSUE THREAD

**Issue by nmcl** [jbosstm/conferences#8]
Use Case 4 is about stress testing but because the script doesn't say what to expect in the output I can only assume that what I see isn't right?

rorschach:devoxxUK2017 marklittle$ mvn exec:java -Pstress
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building Narayana - STM and vertx demo 5.7.2.Final-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- exec-maven-plugin:1.4.0:java (default-cli) @ stm-vertx-demo ---
Waiting for 1000 requests
0 out of 1000 requests failed in 5354 ms
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.605s
[INFO] Finished at: Wed Nov 22 17:26:03 GMT 2017
[INFO] Final Memory: 16M/360M
[INFO] ------------------------------------------------------------------------

**Comment by mmusgrov** [comment]
It is correct:
> Waiting for 1000 requests
> 0 out of 1000 requests failed in 5354 ms
is saying that we made 1000 successful requests ie there were no failed requests.

By the way I miss reported the time (I was out by a factor of 10). It should be 
> 0 out of 1000 requests failed in 535 ms

**Comment by nmcl** [comment]
OK but is that what you'd expect? From the README I expected to see some warnings or failures due to stressing.

**Comment by mmusgrov** [comment]
The intention was to show that by annotating the Theatre object as transactional we can demonstrate that there are no concurrency issues.

I simplified it in the mucon demo where I ran the stress test against two versions of the object (one without STM and one with STM). I just incremented a booking counter. I ran the stress test on the non STM version and showed that the final booking count was less than expected because of interleaving. Converting it into an STM object showed the correct count.
